src/components/TopNav/NavMenu.js

```diff
@@ -7,19 +7,44 @@ import { Link } from 'react-router-dom'
 import NavigationMenu from 'material-ui/svg-icons/navigation/menu'
 import IconButton from 'material-ui/IconButton'
 import * as auth from 'services/authNService'
+import { _ } from 'underscore'
 
-export const NavMenu = ({ dispatch }) =>
-  <IconMenu
+const Tickets = () => {
+  var persistedTickets  = JSON.parse(localStorage['persist:ticket'])
+  var currentTicketId   = window.location.pathname.match(/(\d+)/)[1]
 
+  var idIsNumericalAndNotCurrentTicket  = (id) => /\d+/.test(id) && id != currentTicketId
+  var addCurrentPageToHistory           = () => { history.push(this.to) }
+  var transformIdToTicketLink           = (id) =>
+    <MenuItem
+      key={'ticket-' + id}
+      primaryText={
+        <Link
+          to={'/tickets/' + id}
+          onClick={ addCurrentPageToHistory }>
+          Ticket {id}
+        </Link>
+      }
+    />
+
+  var ticketIds = _.select(Object.keys(persistedTickets), idIsNumericalAndNotCurrentTicket)
+
+  return ticketIds.map(transformIdToTicketLink)
+}
+
+export const NavMenu = (dispatch) => {
```

Comment:
({dispatch}) => (dispatch) is a breaking change. React passes three arguments and the 'arguments' on the <NavMenu dispatch={dispatch} secondArg={secondValue} /> syntax are passed as properties on a single object passed to the component as the first argument: { dispatch:dispatch, secondArg:secondValue }. Removing the {} means NavMenu will receive a 'dispatch' object with a dispatch property whose value we want to use as the dispatch on line 42.